sites/geohub/src/components/controls/VectorPolygonAdvanced.svelte

```diff
@@ -289,7 +311,7 @@
         transition:fade>
         <div class="has-text-centered pb-2">Classification:</div>
         <div class="is-flex is-justify-content-center">
-          <div class="select is-small is-justify-content-center">
+          <div class="select is-normal is-justify-content-center">
```

Comment:
Are you sure bulma has `is-normal` class? I don't think there is such a class name for bulma.

Reply:
sorry, I was wrong. there is `is-normal`
https://bulma.io/documentation/form/select/#sizes